templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends '@W3CWebsiteTemplates/base.html.twig' %}
+
+{% block main %}
+    <h1>{% block title %}{{ page.title|default('Error: {{ status_code }} - {{ status_text }}') }} | W3C{% endblock %}</h1>
+    <p>{{ 'error.intro'|trans({'status_code': status_code, 'status_text': status_text}, 'w3c_website_templates_bundle') }}</p>
```

Comment:
Use component:
```suggestion
    <div class="component component--text">
        <p>{{ 'error.intro'|trans({'status_code': status_code, 'status_text': status_text}, 'w3c_website_templates_bundle') }}</p>
    </div>
```

Reply:
Integrated in https://github.com/w3c/w3c-website-templates-bundle/commit/776d66e897902c8c85388521c3a2561fa634b319